templates/repo/header.tmpl

```diff
@@ -27,7 +27,11 @@
 					</div>
 				</div>
 				{{if $.PullMirror}}
-					<div class="fork-flag">{{ctx.Locale.Tr "repo.mirror_from"}} <a target="_blank" rel="noopener noreferrer" href="{{$.PullMirror.RemoteAddress}}">{{$.PullMirror.RemoteAddress}}</a></div>
+					<div class="fork-flag">
+						{{ctx.Locale.Tr "repo.mirror_from"}}
+						<a target="_blank" rel="noopener noreferrer" href="{{$.PullMirror.RemoteAddress}}">{{$.PullMirror.RemoteAddress}}</a>
+						{{ctx.Locale.Tr "repo.mirror_sync"}} {{TimeSinceUnix .SyncedUnix ctx.Locale}}
```

Comment:
I don't understand your implementation at all.
All this data is already stored.
You simply need to call `{{TimeSinceUnix $.PullMirror.UpdatedUnix ctx.Locale}}`

Reply:
Updated is being set to latest commit date received, not the date of sync itself. At least this is what I got from the code.

Users on the other hand typically want to know mirror they refer to has been recently synced to upstream

Reply:
No, I just had a look at the code, `UpdatedUnix` is set to the time of the sync.

Reply:
That's about setting updated time on a mirror object. Line 546 sets repository update time to the latest commit date (and from the template we access the repository object)

Reply:
Yes, but the pull mirror is also available in the template (see my initial message at the location where I originally commented).
I'm strongly against adding this field to `repository:
That field should belong to a pull mirror (object), not just any repository.
Even more so as you modify the API with your approach, and I don't want to pollute the API with information at the wrong place.

Reply:
Alright, thanks, will take a look